ci-operator/config/openshift/gcp-pd-csi-driver/openshift-gcp-pd-csi-driver-master.yaml

```diff
@@ -39,7 +39,8 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: test
+- always_run: false
```

Comment:
This needs to be updated in 4.13.yaml and 4.14.yaml too

Reply:
Done.